dart.html.markdown

```diff
@@ -303,12 +305,12 @@ example13() {
 
 /// Boolean expressions support implicit conversions and dynamic type
 example14() {
-  var a = true;
+  bool? a = true;
   if (a) {
     print("true, a is $a");
   }
   a = null;
-  if (a) {
+  if (a ?? false) { //if null count as false
```

Comment:
```suggestion
  if (a ?? false) { // if null count as false
```

But this could be worded more clearly.

Reply:
actually, should just delete this example if "Boolean expressions support implicit conversions" is no longer true?